pegasus/sites.v3/code.org/views/teacher_announcements.haml

```diff
@@ -1,28 +1,28 @@
 - cta_url = "http://blog.code.org/post/147243645493/"
 - csp_widgets_url = "http://teacherblog.code.org/post/148059231974/explore-cs-concepts-in-any-classroom-with-these-4"
 
-%div.teacher-announce
-  %a.announce-heading-link{href: 'http://teacherblog.code.org/'}
-    %div.announce-heading
-      = I18n.t(:dashboard_announcements)
-      %div.heading-view-all
-        = I18n.t(:dashboard_announce_view_all)
-        >>
-  %div.announce-content
+.teacher-announce
+  .announce-heading
+    = I18n.t(:dashboard_announcements)
+  .announce-content
     %a{href: cta_url, target: '_blank'}
-      %div.content-img-container{style: 'margin: 15px 50px 0px 50px;'}
+      .content-img-container{style: 'margin: 15px 50px 0px 50px;'}
```

Comment:
Should I just create a new class? Although not likely to be used again and will probably be removed in a week.